client/package.json

```diff
@@ -1,58 +1,43 @@
 {
-  "name": "client",
-  "version": "0.1.0",
+  "name": "vudl-test",
```

Comment:
Should we keep the name as client? Or perhaps use vudl-client? I don't think "test" is appropriate at this stage.